server/application-data/encrypted-odl-key.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove the credential details and have it as an empty string like 
```
{
            "authorization": " "
        }
```

During the application deployment , this shall be included.